product_route_profile/views/product_template.xml

```diff
@@ -11,6 +11,12 @@
             <xpath expr="//field[@name='route_ids']/parent::div" position="attributes">
                 <attribute name="attrs">{'invisible': True}</attribute>
             </xpath>
+            <xpath expr="//field[@name='route_ids']" position="attributes">
+                <attribute name="readonly">1</attribute>
+            </xpath>
+            <xpath expr="//label[@for='route_ids']" position="attributes">
```

Comment:
@bguillot Agree for the first, but should we hide the result routes from profile ?

Reply:
It is already hidden, I only hide the label because it is confusing to have the label but empty